packages/gasket-plugin-dynamic-plugins/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@gasket/plugin-dynamic-plugins",
+  "version": "0.0.0",
+  "description": "Gasket plugin for dynamically adding plugins to Gasket",
+  "type": "module",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib",
+    "cjs"
+  ],
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "require": "./cjs/index.js",
+      "types": "./lib/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict --experimental-vm-modules' jest",
+    "test:watch": "npm run test -- --watch",
+    "test:coverage": "npm run test -- --coverage",
+    "posttest": "npm run lint && npm run typecheck",
+    "typecheck": "tsc",
+    "typecheck:watch": "tsc --watch",
+    "build": "swc lib -d cjs --delete-dir-on-start --strip-leading-paths",
```

Comment:
I really don't want to get into the business of transpiling our plugins. For non-plugin packages, that's fine for now, but let's avoid it for plugins.

What was the need for it here?

Reply:
I saw that there was another esm plugin that transpiled to cjs so I thought that was a pattern we wanted to follow. I can remove it thought if that's something we want to avoid.

Reply:
Which one? Yea let's avoid it

Reply:
I removed the cjs transpilation. The commands plugin is esm and transpiles to cjs

Reply:
Ok cool - I'll take another look.

Ah, thanks for pointing that out about the Command plugin. I spoke with @mmason2-godaddy and we'll get a ticket to remove the transpilation from it. There was a period when plugins were importing the command plugin, so CJS was necessary. That should no longer be the case, and something for use to avoid now.